components/Entry/index.js

```diff
@@ -0,0 +1,35 @@
+import { styled } from "styled-components";
+
+function Entry({ JPDefinition, JPReading, ENDefinition }) {
+  return (
+    <StyledWordCard>
+      <StyledJPDefinition>{JPDefinition}</StyledJPDefinition>
+      <StyledDefinition>[{JPReading}]</StyledDefinition>
+      <StyledDefinition>{ENDefinition.join(" ⦁ ")}</StyledDefinition>
```

Comment:
You can still use a `<ul>`, and use pseudo elements to add the dot in between:

```css
ul :not(:last-child):after {
  content: '⦁';
}
```

[Take inspiration for the pseudo stuff in styled components](https://stackoverflow.com/questions/45871439/before-and-after-pseudo-classes-used-with-styled-components)